src/Console/stubs/middleware.stub

```diff
@@ -10,7 +10,7 @@ use Rebing\GraphQL\Support\Middleware;
 
 class DummyClass extends Middleware
 {
-    public function handle($root, $args, $context, ResolveInfo $info, Closure $next)
+    public function handle($root, array $args, $context, ResolveInfo $info, Closure $next)
```

Comment:
This one is slightly problematic, because this will _override_ the parent (in all the other stub-casses, there's actually no parent method, so it's definitely fine).

If we want to keep it here, we:
- need to update the actual Middleware
- mention it in the changelog
  Though I don't thin it's a breaking change, it's a change we should mention

Reply:
Good catch, I've changed it in \Rebing\GraphQL\Support\Middleware as well and marked it breaking.